03-Bash/kernel_git/kernel_stats.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ ! -d $1  ]]
+then
+	echo "Wrong directory"
+	exit
+fi
+
+cd $1
+
+echo "Quantity of files with extention .c: $(git ls-files '*.c' | wc -l)"
+echo "Quantity of files with extention .cpp: $(git ls-files '*.cpp' | wc -l)"
+echo "Quantity of files with extention .py: $(git ls-files '*.py' | wc -l)"
+
+echo "Quantity of commits with word 'revert' in commit name: $(git log --oneline | grep -c revert | wc -l)"
+
+echo "lines for each author are in *.py files:"
+git ls-files '*.py' | while read f
+do
+	 echo -e "File $f:"; git blame --line-porcelain $f | grep '^author ' | sed 's/^author //' | \
```

Comment:
Here is statistic for each file but it the info about all files together is required.